src/foam/dao/EasyDAO.js

```diff
@@ -566,35 +566,41 @@ return delegate;
       name: 'testData',
       generateJava: false
     },
+    {
+      documentation: 'Enables automated adding of aware doas to qualifying decorator chains',
+      name: 'enableAware',
```

Comment:
This is close enough to `EnabledAware` to be confusing. I think we should try to come up with a better name. `enablePropertyDecorators` maybe?

Reply:
yahh that's probably a good idea; I'm going to change it to enableAwareDecorators